_overviews/psp/overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Overview
+topic: PSP
+layout: overview
+sectionid: overview
+redirect_from:
+  - /overview
+---
+
+## What is the Plugin Server Protocol?
+
+If you are not aware of the Language Server Protocol, please read [this overview]({{ "/overviews/lsp/overview" | prepend: site.baseurl }}) first
+<br>
+<br>
+
+The idea behind the *Plugin Server Protocol (PSP)* is to standardize the protocol between editors (clients) and plugins (servers); so that a single Plugin Server can be re-used in multiple development tools, and tools can support plugins with minimal effort.
+
+PSP is a win for both plugin providers and tools vendors!
+
+## How it works
+
+The Plugin Server Protocol serves as an extension of the existing Language Server Protocol, which standardized how language tooling communicated with editors. Much of the communication and methods are inherited from it, though with some changes:
+
+- New fields in the initialize handshake to subscribe to events that the plugin wishes to receive. A plugin will receive only events it has subscribed to.
+- New methods are added for functionality expected by plugins. (Adding commands to the palette, settings information, starting LSP servers,etc.)
+
+## Capabilities
+
+The PSP inherits the same capabilities defined in the [LSP specification](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/), and adds more that are specific to the protocol.
+
+## Libraries (SDKs) for PSP providers and consumers
+
+To simplify the implementation of language servers and clients, there are libraries or SDKs:
+
+//TODO: link PSP server and client sdks (we mostly won't provide client sdk as it would need decoupling of Lapce's proxy inner workings, but a client sdk definitely needs to be a thing)
```

Comment:
I don't think we should mention that here, and more so, I don't see why it would be important to mention it in the first place, as there is not mention to WASM or WASI in this repository.

Reply:
I can understand that. My reasoning was mostly that since this is currently from-Lapce, there might be confusion that it inherently uses WASM, but I don't think it is significant.